### PR TITLE
Rbkj world creation and enemy

### LINKED ISSUE #9: Implement AI

### LINKED ISSUE #10: Create Door Transition Mechanic

### PR DESCRIPTION
**Pull request by tulza**
closes #9 
closes #10

### CODE REVIEW

File: Scenes/Objects/ghost.gd [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+const AGGROTIME: float = 2.0
+const SPEED : float = 45.0
+var TP_TIMER: float = 10
+
+var target: Node2D
+var aggroTimer: float
+
+var idleTimer: float = 0
+var hasFinishFollowIntoRoom: bool = false
```
Comment (AntGa) [inline Scenes/Objects/ghost.gd:12]: I made a statemachine and state class to inherit from and this is what happens 😭

**Review by AntGa — APPROVED** [review]
LGTM (why is there no statemachines 😭 )

**Comment by tulza** [comment]
> LGTM (why is there no statemachines 😭 )

smile

File: Scenes/Objects/ghost.gd (hunk 1 above)
Comment (tulza) [reply Scenes/Objects/ghost.gd:12]: lol

**Pull request merged**